worker/raft/raftforwarder/worker.go

```diff
@@ -104,11 +111,16 @@ func (w *forwarder) Wait() error {
 
 func (w *forwarder) loop() error {
 	defer w.unsubscribe()
+	if w.config.PrometheusRegisterer != nil {
```

Comment:
We don't need the nil check as a nil registerer is invalid config right?

Reply:
Good point, removing.